### PR TITLE
update node version to a compatible one

### PR DESCRIPTION
**Pull request by dafmdev**
I've been checking heroku issues and when I updated the node version as the base of the image everything worked fine.

### CODE REVIEW

**Review by tesla-srt — APPROVED** [review]

**Comment by feilongfl** [comment]
Sorry for the late reply, I had some problems with my heroku account last month, 
so I didn't get to review this in time.

File: config-heroku.ini [hunk 1]
```diff
@@ -1,12 +1,12 @@
 [General]
 # Instance name can be used to distinguish between different instances using backend api.getInstanceName()
-instanceName="heroku"
```
Comment (feilongfl) [inline config-heroku.ini:3]: I think we should keep the name here.

File: config-heroku.ini [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+instanceName="heroku-dafmdev"
 
 # set to true to allow using Trilium without authentication (makes sense for server build only, desktop build doesn't need password)
-noAuthentication=false
```
Comment (feilongfl) [inline config-heroku.ini:6]: I think `noAuthentication` in public network is not safe.
I would like to know your reasons for doing so

File: config-heroku.ini [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
+noAuthentication=true
 
 # set to true to disable backups (e.g. because of limited space on server)
-noBackup=true
```
Comment (feilongfl) [inline config-heroku.ini:9]: I don't think it makes sense to backup at heroku because I currently haven't found a way to get the backup data back.

**Review by feilongfl — CHANGES_REQUESTED** [review]
I have tested that the upgraded node version can be deployed normally without any major problems.
However, I would like to discuss with you about the configuration file modification.

**Pull request closed without merging**